src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -493,8 +502,34 @@ TEST_F(HMICapabilitiesTest, LoadCapabilitiesFromFile) {
                                                .asInt()));
     }
   }
+}
+
+TEST_F(HMICapabilitiesTest,
+       LoadCapabilitiesFromFile_CheckTextFields_SuccessLoadAndConvert) {
+  SetUpLanguageAndLightCapabilitiesExpectation();
+  hmi_capabilities_->Init(last_state_wrapper_);
+  const auto display_capabilities_so =
+      *(hmi_capabilities_->display_capabilities());
+
+  const uint32_t text_len =
```

Comment:
@ydementieiev Don't use conversion. Method length() returns `size_t`

Reply:
@AByzhynar change to size_t in this [PR](https://github.com/LuxoftSDL/sdl_core/pull/17/commits/53f85366b6a7cfc3fc2ccd60e33886690db075de)